dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/util/DesignTemplateUtil.java

```diff
@@ -254,8 +254,17 @@ private static void addFooterCode(Document templateBody, String headCode){
 	}
 
 	private static String getPageWithValue(Document templateDrawedBody){
-		Element globalContainer = templateDrawedBody.getElementsByAttributeValue(NAME_ATTRIBUTE, MAIN_DIV_NAME_VALUE).get(0);
-		return globalContainer.attr(ID_ATTRIBUTE);
+	    Elements globalContainers = templateDrawedBody.getElementsByAttributeValue(NAME_ATTRIBUTE, MAIN_DIV_NAME_VALUE);
+        if(null!=globalContainers && globalContainers.size()>0){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use isEmpty() to check whether the collection is empty or not. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1155)